cypress/integration/jsPromisesPracticeDOM.spec.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+Cypress.Commands.add('clickButton',
+  (mouseButton) => {
+    cy.get('body').trigger('mousedown', mouseButton)
+      .trigger('mouseup');
+  });
+
+Cypress.Commands.add('checkPromise',
+  (promise) => {
+    cy.get('[data-qa="notification"]').contains(promise);
+  });
+
+const fPromRes = 'First promise was resolved';
+const fPromRej = 'First promise was rejected';
+const sPromRes = 'Second promise was resolved';
+const thPromRes = 'Third promise was resolved';
```

Comment:
these names are too short. Don't shorten words to one character. 

firstResolvedMsg
firstRejectedMsg
...